src/dashboard/Settings/actions.js

```diff
@@ -0,0 +1,53 @@
+/* Actions for the incident */ /* const actions */
+const ADD_NEW_INCIDENT_TYPE = 'INCIDENTS_TYPE:ADD_NEW_INCIDENT_TYPE';
+const SELECTED_INCIDENT_TYPE = 'INCIDENTS_TYPE :SELECTED_INCIDENTS_TYPE ';
+const FILTER_INCIDENTS_TYPE = 'INCIDENTS_TYPE :FILTER_INCIDENT_TYPE';
+const TRIGGER_GET_INCIDENTS_TYPE = 'INCIDENTS_TYPE :TRIGER_GET';
+const STORE_INCIDENTS_TYPE = 'INCIDENTS_TYPE:STORE';
+const UPDATE_INCIDENT_TYPE = 'INCIDENTS_TYPE :UPDATE_INCIDENT_TYPE';
+const COLOR_AUTOFILL = 'INCIDENTS_TYPE :COLOR_AUTOFILL';
+
+/* Actions creater */
+
+const triggerGetIncidentstype = () => ({ type: TRIGGER_GET_INCIDENTS_TYPE });
+const storeIncidents = (incidentsType = []) => ({
+  type: STORE_INCIDENTS_TYPE,
+  incidentsType,
+});
+
+const selectedIncidentType = incidentSelected => ({
+  type: SELECTED_INCIDENT_TYPE,
+  incidentSelected,
+});
+const addNewIncidentType = incidentType => ({
+  type: ADD_NEW_INCIDENT_TYPE,
+  incidentType,
+});
+const updateIncidentType = incidentType => ({
+  type: UPDATE_INCIDENT_TYPE,
+  incidentType,
+});
+const filterIncidentType = filterIcident => ({
+  type: FILTER_INCIDENTS_TYPE,
+  filterIcident,
+});
+const colorAutofill = colorSelected => ({
+  type: COLOR_AUTOFILL,
+  colorSelected,
+});
+export {
```

Comment:
I suggest exporting every action creator/ action type on it's own i.e
```suggestion
export const ACTION_TYPE = 'ACTION_TYPE';
```